transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -95,6 +95,39 @@ pub struct SignedInputRing {
     pub signed_digest: Option<[u8; 32]>,
 }
 
+/// Signature data that is required for any signer to produce signature
+/// and construct transaction
+#[derive(Clone, Debug, Digestible, Eq, PartialEq, Serialize, Deserialize)]
+pub struct SigningData {
```

Comment:
I feel like, this struct could have a better name, because `Data` is not very informative or specific, so it's just like "signing related stuff". I'm going to think about a better name. Not blocking.

Reply:
Agreed, although I failed at coming up with a better name so far.

Reply:
Maybe like `PreparedRctBulletproofs` as contrasted with `SignatureRctBulletproofs`?

The `SignatureRctBulletproofs` name is also weird but not going to change I guess

Reply:
Decided we can deal with names in a followup PR.